src/Compatibility/Core/src/AppHostBuilderExtensions.cs

```diff
@@ -12,13 +12,58 @@ namespace Microsoft.Maui.Controls.Compatibility
 {
 	public static class AppHostBuilderExtensions
 	{
+		public static IAppHostBuilder RegisterCompatibilityRenderers(this IAppHostBuilder builder)
+		{
+			// This won't really be a thing once we have all the handlers built
+			var defaultHandlers = new Dictionary<Type, Type>
+			{
+				{  typeof(IButton), typeof(ButtonHandler) },
+				{  typeof(IEntry), typeof(EntryHandler) },
+				{  typeof(ILayout), typeof(LayoutHandler) },
+				{  typeof(ILabel), typeof(LabelHandler) },
+				{  typeof(ISlider), typeof(SliderHandler) },
+				{  typeof(ISwitch), typeof(SwitchHandler) }
+			};
+
+			Forms.RegisterCompatRenderers(
+				new[] { typeof(RendererToHandlerShim).Assembly } ,
```

Comment:
This makes it so we only can through one assembly for renderers. 

In the final version we'll probably list this all out by hand but for now this works. If this slows down the sample too much then I can type it all out :-) 

@rmarinho